<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayer</class>
 <widget class="QDialog" name="VideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayoutVideo"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>261</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayoutAvancement_2"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>261</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayoutTemps_2">
    <item>
     <widget class="QLCDNumber" name="lcdNumber_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>311</y>
     <width>179</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayoutBoutons_2">
    <item>
     <widget class="QPushButton" name="btnPlay_2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/audio_player_play.png</normaloff>images/audio_player_play.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPause_2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/audio_player_pause.png</normaloff>images/audio_player_pause.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStop_2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/audio_player_stop.png</normaloff>images/audio_player_stop.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>311</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayoutVolume_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/audio_player_volume.png</pixmap>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
